### DISCUSSION TITLE
Github Pages does not publish full set of files from gh-pages branch

### DISCUSSION THREAD

**Discussion by bssrdf** [community/community#40164] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, I am trying to publish a simple React app to github pages. There is a single index.html file and a build dir containing some supporting  js and css files.  I created a gh-pages branch whose root dir contains the html and build/

The deploy process went ok.  However, when I access the pages, only index.html is loaded. All the supporting js and css in build dir are missing. 

This is the [repo](https://github.com/bssrdf/lifequote)  and its pages website is [https://bssrdf.github.io/lifequote/](https://bssrdf.github.io/lifequote/)

Please help

Thanks,

**Comment by radar-blip** [comment]
` <link rel="stylesheet" href="/build/style.css">`

Where is your build folder?

**Reply by bssrdf** [reply]
The build folder is right there with index.html. This the [branch](https://github.com/bssrdf/lifequote/tree/gh-pages) for both index.html and build folder. For some reason, only index.html was published.

**Reply by radar-blip** [reply]
Ok, guess I didn't see it before. Thanks!

**Reply by bssrdf** [reply]
Thanks for the help.

Here is  what chrome sees when visiting the pages website. If everything works properly, there should be a build under 'lifequote'. Instead, a build folder appeared under bssrdf.github.io domain. 
 
![ghpage](https://user-images.githubusercontent.com/689043/204099933-babd1279-96ae-48df-bcbd-1f4c58c1e3d5.png)

**Comment by bssrdf** [comment, marked as answer]
Finally I figured it out.  The ref to build/* are wrong. 
After changing from
```
<link rel="stylesheet" href="/build/style.css">
```

to 
```
<link rel="stylesheet" href="build/style.css">
```
and republishing, the pages website works fine.

**Reply by masayukipy** [reply]
Perfect!

**Reply by radar-blip** [reply]
Awesome! Glad you got it figured out!
